www/manifest.json

```diff
@@ -1,19 +1,19 @@
-{

```

Comment:
maybe whitespace difference, if you compare with master do you get a difference?